sklearn/metrics/tests/test_pairwise.py

```diff
@@ -78,7 +79,7 @@ def test_pairwise_distances():
     S3 = manhattan_distances(X, Y, size_threshold=10)
     assert_array_almost_equal(S, S3)
     # Test cosine as a string metric versus cosine callable
-    # "cosine" uses sklearn metric, cosine (function) is scipy.spatial
+    # "cosine" uses scikit-learn metric, cosine (function) is scipy.spatial
```

Comment:
Glad to see I'm not alone in being on the fence about that one. I think it'd probably be most clear as ` The string "cosine" uses sklearn.metric, while the function cosine is scipy.spatial`. 

I'll update that in a moment.